### ISSUE TITLE
Improve documentation to include metadata on queries

### ISSUE THREAD

**Issue by alexpaden** [asg017/sqlite-vec#50]
I think this would be a big improvement with low effort

**Comment by lojik-ng** [comment]
Agreed! I created a normal table to hold my extra data in [https://github.com/lojik-ng/sqlite-vec-example](https://github.com/lojik-ng/sqlite-vec-example)
